scnn.py

```diff
@@ -130,8 +130,8 @@ def backward(ctx, grad_output):
                                                      padding, dilation, groups)  
             else:
```

Comment:
We can potentially remove the if/else statement here as well. Now that this has been fixed.

Reply:
I removed the if/else statement, see commit 6b86db516f11c61a66a1df4f94a47f78058c194c